packages/workbox-window/src/Workbox.ts

```diff
@@ -281,21 +282,21 @@ class Workbox extends WorkboxEventTarget {
    * @param {Object} data An object to send to the service worker
    * @return {Promise<Object>}
    */
-  async messageSW(data: object) {
+  async messageSW(data: Record<string, unknown>): Promise<any> {
```

Comment:
Just wanted to check that this transition from `object` to `Record<string, unknown>` is backwards-compatible.